Cartfile

```diff
@@ -1,2 +1,2 @@
 github "raphaelmor/Polyline" == 5.0.3
-github "mapbox/turf-swift" ~> 2.4
```

Comment:
Unfortunately Carthage resolves both `~> 2.4` and `~> 2.4.0` to the latest `v2.6.1` version of Turf. Since Mapbox Directions brings iOS 11 removal only in `v2.8.0` we have to pin to the latest version of Turf that precedes `v2.6.1`.

Reply:
Correct, Carthage [intentionally does not support](https://github.com/Carthage/Carthage/blob/a91d086ceaffef65c4a4a761108f3f32c519940c/Documentation/Artifacts.md#version-requirement) an “up to next minor version” operator, because it considers that semantic to violate the Semantic Versioning standard. It only makes an exception for pre-v1.0 versions.